build.py

```diff
@@ -51,11 +62,18 @@ def build(build_directory, node_registry_url, clean=False):
         help="node registry URL",
     )
 
+    parser.add_argument(
+        "-g",
+        "--config-file",
+        default=os.path.join(THIS_DIR, "config.toml.example"),
```

Comment:
config.toml (the default is not the example file)

Reply:
Added "config.toml" to gitignore.